docs/infrastructure/workers/dynamic-worker-pools/ubuntu-1804-end-of-life.md

```diff
@@ -0,0 +1,76 @@
+﻿---
+title: Ubuntu 18.04 End-of-life
+description: Describing the deprecation process of Ubuntu 18.04 Dynamic Workers.
+position: 50
+hideInThisSection: true
+hideInThisSectionHeader: true
+---
+
+Our Ubuntu Dynamic Workers are being upgraded to use Ubuntu 22.04, this upgrade will result in breaking changes for users of gcloud CLI and users of .NET Core 2.1/3.1 and Ubuntu 18.04 capabilities that are not offered by the updated replacements.
```

Comment:
It looks like we're using lower case `dynamic workers` everywhere - let's make it consistent then